src/app/containers/App/App.test.jsx

```diff
@@ -11,11 +11,21 @@ describe('App', () => {
   let wrapper;
   let setStateSpy;
   let loadInitialDataSpy;
-  const initialData = { data: 'Some initial data' };
+  const initialData = { pageData: 'Some initial data' };
+  const error = 'Error!';
+  const match = { params: { service: 'news', amp: false } };
+
+  const route = {
+    getInitialData: jest.fn().mockImplementation(() => Promise.reject(error)),
+  };
+
+  reactRouterConfig.matchRoutes = jest.fn().mockReturnValue([{ route, match }]);
 
   reactRouterConfig.renderRoutes = jest
     .fn()
-    .mockReturnValue(<h1>{initialData.data}</h1>);
+    .mockReturnValue(<h1>{initialData.pageData}</h1>);
+
+  // const getRouteProps = jest.fn().mockReturnValue(match);
```

Comment:
Really tiny, is this still needed or needed in a future refactor?

Reply:
oops, No this can go. Thanks.